src/material/bottom-sheet/bottom-sheet.md

```diff
@@ -96,20 +96,49 @@ for `MAT_BOTTOM_SHEET_DEFAULT_OPTIONS` in your application's root module.
 
 
 ### Accessibility
-By default, the bottom sheet has `role="dialog"` on the root element and can be labelled using the
-`ariaLabel` property on the `MatBottomSheetConfig`.
 
-When a bottom sheet is opened, it will move focus to the first focusable element that it can find.
-In order to prevent users from tabbing into elements in the background, the Material bottom sheet
-uses a [focus trap](https://material.angular.io/cdk/a11y/overview#focustrap) to contain focus
-within itself. Once a bottom sheet is closed, it will return focus to the element that was focused
-before it was opened.
+`MatBottomSheet` creates modal dialogs that implement the ARIA `role="dialog"` pattern. This root
+dialog element should be given an accessible label via the `ariaLabel` property of
+`MatBottomSheetConfig`.
+
+#### Keyboard interaction
+By default, the escape key closes `MatBottomSheet`. While you can disable this behavior via
```

Comment:
```suggestion
By default, the escape key closes `MatBottomSheet`. While you can disable this behavior via
```
Remove "via". Sentence should read, "While you can disable this behavior by using..."